content/news/XXXX-XX-XX-mixxx-2-3-4-released.md

```diff
@@ -0,0 +1,32 @@
+title: Mixxx 2.3.4 Released
+authors: Nikolaus Einhauser
+status: draft
+tags: 2.3, 2.3.4, release announcement
+comments: yes
+
+
+You know the drill, another 2.3 bugfix release. [Mixxx
+2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.3) to be exact.
```

Comment:
```suggestion
2.3.4](https://github.com/mixxxdj/mixxx/releases/tag/2.3.4) to be exact.
```

Reply:
Ah, copy-paste mistake, thank you. Fixed.